### PR TITLE
fix: checkout -> check out

### PR DESCRIPTION
**Pull request by rhcarvalho**
And a few more improvements in surrounding text.

### CODE REVIEW

**Review by HazAT — APPROVED** [review]

File: src/docs/accounts/quotas/manage-event-stream-guide.mdx [hunk 1]
```diff
@@ -31,7 +31,7 @@ The integration is enabled by default and adds the following configuration optio
 - `denyUrls`: A list of strings or regex patterns that match error URLs which should be blocked from sending events (Configuring both options on the SDK can be used to block **subdomains** of the domains listed in `allowUrls`).
 - `ignoreErrors`: Instruct the SDK to never send an error to Sentry if it matches any of the listed error **messages**. If no message is available, the SDK will try to compare against an underlying **exception type and value**.
 
-For more information and code samples checkout:
+For more information and code samples check out:
 
 - [Integrating the SDK: Decluttering Sentry](/platforms/javascript/#decluttering-sentry)
```
Comment (PeloWriter) [inline src/docs/accounts/quotas/manage-event-stream-guide.mdx:36]:
```suggestion
- [Integrating the SDK: Filtering Events Reported to Sentry](/platforms/javascript/config/filter/)
```

File: src/docs/accounts/quotas/manage-event-stream-guide.mdx [hunk 2]
```diff
@@ -31,7 +31,7 @@ The integration is enabled by default and adds the following configuration optio
...
 - [Filtering Events](/error-reporting/configuration/filtering)
```
Comment (PeloWriter) [inline src/docs/accounts/quotas/manage-event-stream-guide.mdx:37]: the new link covers this content, so I'd remove this one.

File: src/docs/accounts/quotas/manage-event-stream-guide.mdx [hunk 3]
```diff
@@ -41,7 +41,7 @@ For more information and code samples checkout:
 
 The integration attaches global handlers to capture uncaught exceptions (`onerror`) and unhandled rejections (`onunhandledrejection`). Both handlers are enabled by default but can be disabled through configuration. For more information see [GlobalHandlers Integration](/platforms/javascript/#globalhandlers)
 
-Checkout additional configuration options with the [TryCatch](/platforms/javascript/#trycatch) and [ReportingObserver](/platforms/javascript/#reportingobserver) integrations.
+Check out additional configuration options with the [TryCatch](/platforms/javascript/#trycatch) and [ReportingObserver](/platforms/javascript/#reportingobserver) integrations.
```
Comment (PeloWriter) [inline src/docs/accounts/quotas/manage-event-stream-guide.mdx:44]: oof. I see that we're missing the heading for TryCatch in /platforms/javascript/integrations/default/. I will fix this. Suggest we link to the page for now.
```suggestion
Check out additional configuration options with the [TryCatch](/platforms/javascript/integrations/default/) and [ReportingObserver](/platforms/javascript/integrations/plugin/#reportingobserver) integrations.
```

File: src/docs/clients/cocoa/migration.mdx [hunk 4]
```diff
@@ -21,15 +23,19 @@ tags: []
...
+Remove `KSCrash`, as it is now bundled in the Sentry SDK.
 
   ```ruby
   github "getsentry/sentry-cocoa" "3"
   ```
 
-- Public API We changed a lot of the public API, please checkout [Advanced Usage](/clients/cocoa/advanced/#advanced) for more examples about that.
+### Public API
+
+We changed a lot of the public API, please check out [Advanced Usage](/clients/cocoa/advanced/#advanced) for more examples about that.
```
Comment (PeloWriter) [inline src/docs/clients/cocoa/migration.mdx:36]:
```suggestion
Please check out our public API [Advanced Usage](/clients/cocoa/advanced/#advanced) for more examples.
```

File: src/docs/clients/cocoa/migration.mdx [hunk 5]
```diff
@@ -21,15 +23,19 @@ tags: []
...
 
 ## Upgrade from 3.x.x to 4.x.x {#upgrade-from-3-x-x-to-4-x-x}
 
-- We removed the external KSCrash dependency and migrated the code directly into our SDK.
-  There shouldn't be any conflicts including KSCrash it's just not necessary anymore.
+- We removed the external `KSCrash` dependency and migrated the code directly into our SDK.
+  There shouldn't be any conflicts including `KSCrash`, as it's just not necessary anymore.
```
Comment (PeloWriter) [inline src/docs/clients/cocoa/migration.mdx:41]:
```suggestion
  There shouldn't be any conflicts including `KSCrash`, as it's no longer necessary.
```

File: src/docs/clients/javascript/integrations.mdx [hunk 6]
```diff
@@ -11,7 +11,9 @@ Integrations extend the functionality of Raven.js to cover common libraries and
 
 ## What are plugins? {#what-are-plugins}
 
-In Raven.js, plugins are little snippets of code to augment functionality for a specific application/framework. It is highly recommended to checkout the list of plugins and use what apply to your project.
+In Raven.js, plugins are little snippets of code to augment functionality for a
+specific application/framework. It is highly recommended to check out the list
```
Comment (PeloWriter) [inline src/docs/clients/javascript/integrations.mdx:15]:
```suggestion
specific application/framework. We highly recommended you check out the list
```

**Comment by rhcarvalho** [comment]
@PeloWriter thanks for the suggestions. Since they're tangential to the change here, I'll get this merged and apply your proposals in a follow up.

**Pull request merged**